lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -393,12 +393,16 @@ flt-glass-pane * {
     // is 1.0.
     window.debugOverrideDevicePixelRatio(1.0);
 
-    if (browserEngine == BrowserEngine.webkit) {
+    if (html.window.visualViewport == null &&
```

Comment:
Let's leave a comment explaining the possibilities that trigger the `if` block, including Firefox.

Reply:
done.